Add time-tracking extension

EinLinuus opened the pull request:
## Description

<!-- A summary of your change. If you add a new extension or command, explain what it does. -->

## Screencast

<!-- If you add a new extension or command, include a screencast (or screenshot for straightforward changes). A good screencast will make the review much faster - especially if your extension requires registration in other services.  -->

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

pernielsentikaer reviewed (changes requested):
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

- `Start Timer` and `Stop Timer` should both be [no-view commands](https://developers.raycast.com/information/lifecycle#no-view-command) instead.

- `View Project Timers` has [Empty State Flicker](https://developers.raycast.com/basics/prepare-an-extension-for-store#empty-states)

- Could be nice with icons in the ActionPanel

![time-tracking 2024-04-12 at 10 01 38](https://github.com/raycast/extensions/assets/38033107/97412cfc-096b-4170-9d69-92b6f907a204)

- Maybe add a [confirmAlert](https://developers.raycast.com/api-reference/feedback/alert#confirmalert) with `rememberUserChoice` when deleting entries

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

EinLinuus commented:
Hi, thank you for your feedback!

I've changed the following:
- start & stop commands are now no-view commands
- the list command now has a loading state
- actions now have icons
- added a confirmAlert to the delete action

In addition to the changes requested, I also added a "Copy Duration" action to the list.

pernielsentikaer reviewed (approved):
Hi 👋 

Looks good to me, approved 🔥
